### ISSUE TITLE
Can't install contribflow, either as sudo (globally) or locally into a project

### ISSUE THREAD

**Issue by jrw95** [brightcove-archive/zencoder_contribflow#5]
I tried to run the following as sudo, and got the following error:

``` bash
npm install -g contribflow

npm ERR! Error: ENOENT, chmod '/usr/local/lib/node_modules/contribflow/bin/contrib.js'
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>

npm ERR! System Darwin 12.3.0
npm ERR! command "node" "/usr/local/bin/npm" "install" "-g" "contribflow"
npm ERR! cwd /Users/jwhisenant/work/witmtn/video-js
npm ERR! node -v v0.8.22
npm ERR! npm -v 1.2.14
npm ERR! path /usr/local/lib/node_modules/contribflow/bin/contrib.js
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/jwhisenant/work/witmtn/video-js/npm-debug.log
npm ERR! not ok code 0
```

Assuming that there was a problem writing to /usr/local/lib (even as sudo), I tried to install contribflow locally into a project:

``` bash
npm install contribflow
npm ERR! Error: ENOENT, chmod '/Users/jwhisenant/work/witmtn/video-js/node_modules/contribflow/bin/contrib.js'
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>

npm ERR! System Darwin 12.3.0
npm ERR! command "node" "/usr/local/bin/npm" "install" "contribflow"
npm ERR! cwd /Users/jwhisenant/work/witmtn/video-js
npm ERR! node -v v0.8.22
npm ERR! npm -v 1.2.14
npm ERR! path /Users/jwhisenant/work/witmtn/video-js/node_modules/contribflow/bin/contrib.js
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/jwhisenant/work/witmtn/video-js/npm-debug.log
npm ERR! not ok code 0
```

I can't upload the entire npm_debug_log files, because github doesn't support that file type.  Go figure.

**Comment by heff** [comment]
Ok, it looks like I had broken the pointer to the bin file in the package.json. It should be fixed now in 0.3.3

**Comment by jrw95** [comment]
Closing, I was able to get this installed.
To install globally, you need to use sudo.
